src/main/java/seedu/duke/Finux.java

```diff
@@ -0,0 +1,85 @@
+package seedu.duke;
+
+import seedu.duke.command.CreditScoreMap;
+import seedu.duke.command.Command;
+import seedu.duke.command.CommandHandler;
+import seedu.duke.command.ExitCommand;
+import seedu.duke.exception.FileLoadingException;
+import seedu.duke.parser.ParserHandler;
+import seedu.duke.record.RecordList;
+import seedu.duke.storage.Storage;
+import seedu.duke.ui.Ui;
+
+import java.util.ArrayList;
+
+import static seedu.duke.common.Constant.FINUX_LOGGER;
+
+public class Finux {
+    private Ui ui;
+    private RecordList records;
+    private Storage storage;
+    private CommandHandler commandHandler;
+    private ParserHandler parserHandler;
+    private CreditScoreMap creditScoreMap;
+
+    /**
+     * Main entry-point for Finux application.
+     */
+    public static void main(String[] args) {
+        new Finux().run();
+    }
+
+    /**
+     * Runner for the FINUX Application.
+     */
+    private void run() {
+        start();
+        commandLooper();
+        end();
+    }
+
+    /**
+     * Starts the main application.
+     */
+    private void start() {
+        try {
+            ui = new Ui();
+            storage = new Storage();
+            parserHandler = new ParserHandler();
+            commandHandler = new CommandHandler();
+            storage.loadFile();
+            records = new RecordList(storage.getRecordListData());
+            creditScoreMap = new CreditScoreMap(
+                    storage.getCreditScoreHashMapData());
+            ui.printWelcomeMessage();
+        } catch (FileLoadingException e) {
+            Ui.printInitError();
+            FINUX_LOGGER.logWarning("Unable to load finux.txt!");
+            System.exit(-1);
+        }
+    }
+
+    /**
+     * Loops the application until an EXIT command is parsed.
+     */
+    private void commandLooper() {
+        while (!commandHandler.isExit()) {
+            String rawInput = ui.getUserInput();
```

Comment:
1. Consider to add the return type `String` for this like the other parts below for consistency.
2. Method arguments need to be updated to match implementation. similarly for `execute()`.
![Presentation1](https://user-images.githubusercontent.com/19249948/113077719-4334f280-9204-11eb-9342-3661f823de6b.png)

Reply:
Thanks for pointing this out! Will rectify the errors.
We are still in the process of standardizing our diagrams.